src/simulation/m_start_up.fpp

```diff
@@ -1058,6 +1058,10 @@ contains
                 t_step - t_step_start + 1, &
                 t_step_stop - t_step_start + 1, &
                 t_step
+                if (relax) then
+                    !$acc update host(max_iter_pc_ts)
```

Comment:
This looks like you are debugging your feature. As far as I can see, `max_iter_pc_ts` is constant throughout the simulation so we shouldn't need to update the host copy and print its value to stdout every iteration. Am I missing something?

Reply:
This is not debugging. I am outputting max_iter_pc_ts into the ".out" file. Look at the image attached for reference

![image](https://github.com/MFlowCode/MFC/assets/22774282/150b72e7-da7f-4097-8e54-a23eadb64305)